pisound-btn/pisound-btn.c

```diff
@@ -126,7 +124,7 @@ static bool read_line(FILE *f, char *buffer, size_t n)
 	return true;
 }
 
-static void read_config_value(const char *conf, const char *value_name, char *dst, char *args, size_t n, const char *default_value)
+static bool read_config_value(const char *conf, const char *value_name, char *dst, char *args, size_t n, const char *default_value)
```

Comment:
Please use an int "errno" value instead of a bool, it gives possibility to provide more insight into what went wrong, like there was an invalid argument, or there was some I/O error. Same for get_action_name and get_action_script